app/src/main/res/layout-land/back_arrow_toolbar.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.appcompat.widget.Toolbar xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  android:id="@+id/back_arrow_toolbar"
+  android:layout_width="match_parent"
+  android:layout_height="56dp"
+  android:background="?attr/colorPrimary"
+  android:minHeight="?attr/actionBarSize"
+  android:textSize="20sp"
```

Comment:
Remove text size and add this style for the toolbar
  `app:titleTextAppearance="@style/ToolbarTextAppearance"` make this change in portrait as well. Also change minHeight to 56dp.